pages/index.js

```diff
@@ -1,7 +1,187 @@
+						imageSrc='https://picsum.photos/200/300'
+						imageAlt='#'
+						text='Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras
+							fringilla facilisis tortor non eleifend. Ut.'
+						link='#'
+						title='Ui/UX Design'
+					/>
+				</div>
+				<div className='index--third'>
+					<Triangle></Triangle>
+					<h3 className='index--portfolio'>Portfolio</h3>
+					<h2>Check Our Latest Cases</h2>
+					<Link href='#'>
+						<a>View All ➡️</a>
+					</Link>
+					<Image
+						src='https://picsum.photos/200/300'
+						width='200'
+						height='300'
+						alt='rando'
+					></Image>
+					<Image
+						src='https://picsum.photos/200/300'
+						width='200'
+						height='300'
+						alt='rando'
+					></Image>
+					<Image
+						src='https://picsum.photos/200/300'
+						width='200'
+						height='300'
+						alt='rando'
+					></Image>
+				</div>
+				<div className='index--fourth'>
+					<h2 className='cases'>CASES</h2>
+					<h3>cases</h3>
+					<h2>Always be in touch</h2>
+					<label htmlFor='email'>email</label>
+					<input type='email' id='email' />
+					<label htmlFor='checkbox'>I accept all Terms and Conditions</label>
+					<input type='checkbox' id='checkbox' />
+				</div>
+				<div className='fifth'>
+					<h1>DIGITAL</h1>
+					<h2>DIGITAL</h2>
+				</div>
+				<div className='index--sixth'>
+					<div className='index-whatwedo'>
+						<Triangle />
+						<h3 className='index--digital__flavortxt'>What We Do</h3>
+						<p>
+							Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras
+							fringilla facilisis tortor non eleifend. Ut.
+						</p>
+					</div>
+					<div className='index--wwditems'>
+						<WhatWeDo
+							title='Branding'
+							text='Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras
+							fringilla facilisis tortor non eleifend. Ut.'
+							image='https://picsum.photos/200/300'
+							imageAlt='alt'
+						/>
+						<WhatWeDo
+							title='Strategy'
+							text='Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras
+							fringilla facilisis tortor non eleifend. Ut.'
+							image='https://picsum.photos/200/300'
+							imageAlt='alt'
+						/>
+						<WhatWeDo
+							title='ECommerce'
+							text='Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras
+							fringilla facilisis tortor non eleifend. Ut.'
+							image='https://picsum.photos/200/300'
+							imageAlt='alt'
+						/>
+						<WhatWeDo
+							title='Support'
+							text='Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras
+							fringilla facilisis tortor non eleifend. Ut.'
+							image='https://picsum.photos/200/300'
+							imageAlt='alt'
+						/>
+					</div>
+				</div>
+				<div className='index--seventh'>
+					<div className='index--bloghead'>
+						<Triangle />
+						<h3 className='index--digital__flavortxt'>Blog</h3>
+						<h2>The Latest News In Digital</h2>
+						<div>READ BLOG ➡️</div>
+					</div>
+					<div className='index--blogbody'>
+						<Blog
```

Comment:
You have a 'Blog' component `pages/blog.js` To avoid confusion you might want to consider renaming this component.